Modules/_datetimemodule.c

```diff
@@ -6516,6 +6553,17 @@ PyInit__datetime(void)
     PyModule_AddIntMacro(m, MINYEAR);
     PyModule_AddIntMacro(m, MAXYEAR);
 
+    /* IsoCalendarDate */
+    if (!initialized) {
```

Comment:
In existing uses of structseq, I see both this initialization idiom used as well as checking whether `<type object>.tp_name == NULL`. The latter avoids using a module-level static variable, keeping the relevant logic all in one place. Would someone like to weigh in on which is preferred here?

Reply:
> Would someone like to weigh in on which is preferred here?

Don't really care.  As a (very) old-timer here, I recognize the intent of the module-level static instantly, because that's how it was done from the start.  But the other way is fine too, provided there's a comment clarifying the intent of the otherwise-weird-looking test.